WolvenKit.App/ViewModels/Shell/ChunkViewModel.cs

```diff
@@ -196,6 +201,11 @@ public partial class ChunkViewModel : ObservableObject, ISelectableTreeViewItemM
     {
         if (IsExpanded && !_propertiesLoaded)
             CalculateProperties();
+
+        if (IsShiftBeingHeld)
+        {
+            _chunkViewModelTools.SetChildExpansionStates(this, IsExpanded);
```

Comment:
this.SetChildExpansionStates(IsExpanded);

Reply:
Is it enough to have <ClassName>Extension.cs to have it immediately available?

Reply:
doesn't depend on the name, it is available when you have a static method with this as first argument for that class

Reply:
Ahh, yeah - I wanted to make it a static class, but that would make HotReload not work for it, so I instantiated it. Any advice about that?

Reply:
huh, I didn't know that... no, no idea. that's weird but okay we can leave it nonstatic then I guess

Reply:
I don't want to instantiate this for every single tree view node that I'm touching >.< I'll check later if I can make a factory class or something